src/app/zwiazek/kalendarz/_components/Events/List.tsx

```diff
@@ -0,0 +1,460 @@
+interface Card {
+  id: string;
+  title: string;
+  image: {
+    src: string;
+    alt: string;
+  };
+  date: string;
+  description: string;
+}
+
+export const historicalEventList: Card[] = [
```

Comment:
This file should be named `data` or something like that as it contains data. Capitalized files with TSX extension indicate that it contains component, while it's not